src/_filter/string/title-case.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @ngdoc filter
+ * @name title-case
+ * @kind function
+ *
+ * @description
+ * get string with {n} and replace match with enumeration values
+ */
+angular.module('a8m.title-case', [])
+  .filter('titleCase', function () {
```

Comment:
What do you think about changing the name to `titleize`? I want this module to be aligned with [ng-pipes](https://github.com/a8m/ng-pipes/blob/master/src/string/titleize.pipe.ts)